etc/DependencyInstaller.sh

```diff
@@ -105,12 +111,17 @@ _installCommonDev() {
         cd "${baseDir}"
         git clone -b "v${spdlogVersion}" https://github.com/gabime/spdlog.git
         cd spdlog
-        cmake -B build .
-        cmake --build build -j $(nproc) --target install
+        ${cmakePrefix}/bin/cmake -B build .
+        ${cmakePrefix}/bin/cmake --build build -j $(nproc) --target install
     else
         echo "spdlog already installed."
     fi
 
+    if [[ ! -z ${PREFIX} ]]; then 
+        export PATH=${PREFIX}/bin:$PATH
+        echo "export PATH=${PREFIX}/bin:$PATH" >> ~/.bash_profile
+    fi
```

Comment:
I don't know if we should modify the users env. Maybe just a suggestion at the end of the script is good enough. If a  user runs this multiple times it can pollute their `bash_profile`. @rovinski @maliberty thoughts? I think this is the policy of homebrew on mac, after installing a package it recommends:
```
Run:
   echo export PATH=${PREFIX}/bin:'$PATH' >> ~/.bash_profile
```
Also note the use of single quotes to let the value of PATH be resolved when sourcing the bash_profile and not when running this script.